<launch>

  <!--	Arguments	 -->
  <arg name="save_image_raw" default="false"/>
  <arg name="save_skeleton_message" default="false"/>
  <arg name="save_skeleton_pose" default="true"/>
  <arg name="data_filtering" default="true"/>
  <arg name="ignore_leg_markers" default="true"/>

	<!--	Node	-->
	<node pkg="gesture_recognition_nuitrack" type="data_acquisition_Node" name="data_acquisition_Node" output="screen">
    <param name="save_image_raw" value="$(arg save_image_raw)"/>
    <param name="save_skeleton_message" value="$(arg save_skeleton_message)"/>
    <param name="save_skeleton_pose" value="$(arg save_skeleton_pose)"/>
    <param name="data_filtering" value="$(arg data_filtering)"/>
    <param name="ignore_leg_markers" value="$(arg ignore_leg_markers)"/>
  </node>

</launch>
